app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx

```diff
@@ -14,6 +14,12 @@ const ButtonText = styled.Text<{ active: boolean }>(({ theme, active }) => ({
   fontSize: 11,
 }));
 
+const styles = StyleSheet.create({
+  focussedButton: {
```

Comment:
just a nit pick but I guess it should be focusedButton

Reply:
Oh sure, I missed that one 😅